modules/core/client/config/core.client.routes.js

```diff
@@ -15,28 +19,52 @@ angular.module('core').config(['$stateProvider', '$urlRouterProvider',
     $stateProvider
     .state('home', {
       url: '/',
-      templateUrl: 'modules/core/client/views/home.client.view.html'
+      views: {
+        'hero': {
+          templateUrl: 'modules/core/client/views/hero.client.view.html'
+        },
+        'main': {
+          controller: 'HomeController',
```

Comment:
@trainerbill We're already using controllerAs in the Articles module. https://github.com/meanjs/mean/blob/master/modules/articles/client/config/articles.client.routes.js#L21

Since the merging of the Styleguide implementation, I believe we want to include the styleguide in all PR's going forward, if it makes sense; i.e. the client-side routing/templates are getting modified.